src/object.d

```diff
@@ -3583,3 +3596,21 @@ unittest
     scope arr = [S(&p)];
     auto a = arr.dup; // dup does escape
 }
+
+unittest
+{
+    // Make sure basic stuff works with future Throwable.message
+    class NoMessage : Throwable
+    {
+    }
+    auto nm = new NoMessage;
+
+    class WithMessage : Throwable
+    {
+        override const(char)[] message() const
```

Comment:
I suggest one more class with `const(char)[] message() const` (without `override` - it should still work modulo deprecation). If you put this into `test/` suite you can inspect the stderr output and make sure the deprecation is triggered.